mopro-ffi/tests/bindings/test_mopro.swift

```diff
@@ -6,51 +6,26 @@ let moproCircom = MoproCircom()
 let wasmPath = "./../../../../mopro-core/examples/circom/multiplier2/target/multiplier2_js/multiplier2.wasm"
 let r1csPath = "./../../../../mopro-core/examples/circom/multiplier2/target/multiplier2.r1cs"
 
-// TODO: should handle 254-bit input
-func serializeOutputs(_ int32Array: [Int32]) -> [UInt8] {
```

Comment:
I imagine we have functions like

`func serializeInputs(_ int32Array: [Int32]) -> [String]`

// TODO: Add proper BigNumber library
// For now can mock with `struct BigNumber` and then `BigNumber("1532423423432")`

`func serializeInputs(_ BNArray: [BigNumber]) -> [UInt8]`

Since Outputs and Inputs are symmetrical, and there's really no "output", perhaps we can just call this serializeInputs? If we think it is confusing to not have a serializeOutputs method we can just have it be identical / alias to serializeInputs.

Then similarly for deserialization

`func deserializeInputs(_ strArray: [String]) -> [BigNumber]`

That way the fact that we serialize to strings is an implementation detail (this could change in the future) and we can easily add serialization methods for various types

Use of BigNumber in Swift seems like a separate issue and more of an app concern for now, so just mocking this with a basic type wrapping string in app/tests seems OK for now

Reply:
I think we can compute over BigNumber before calling `generateProof`
(there should be a `toString` like function in such library)
but if we can handle `serialize_inputs` and `deserialize_inputs` in rust it could be more easier for swift developers?
I prefer not to have these two functions `serializeInputs` and `deserializeInputs` in swift